lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -57,6 +55,17 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :name, _("Name")
           field :label, _("Label")
         end
+
+        field :filters_applied, _("Has Applied Filters")
+        collection :applied_filters, _("Filters Applied") do
+          collection :applied_filters, _("") do
+            from :filter do
+              field :name, _("Name")
+              field :id, _("Id")
```

Comment:
We always (at least I hope) show `id` first.
```suggestion
              field :id, _("Id")
              field :name, _("Name")
```